### PR TITLE
Refactoring SegmentCfgControllerTests done

### PR DESCRIPTION
**Pull request by A9afon**
Refactoring SegmentCfgControllerTests done

### CODE REVIEW

File: src/test/java/com/a5lab/axion/domain/segment/SegmentCfgControllerTests.java [hunk 1]
```diff
@@ -23,172 +23,210 @@
...
+                .andExpect(status().isOk())
+                .andExpect(view().name("settings/segments/index"))
+                .andExpect(model().attributeExists("segmentDtoPage"))
+                .andExpect(model().attributeExists("pageNumbers"))
+                .andReturn();
+        String content = result.getResponse().getContentAsString();
+
+        Assertions.assertTrue(content.contains(segmentDto.getTitle()));
+        Assertions.assertTrue(content.contains(segmentDto.getDescription()));
+
```
Comment (arufanov) [inline src/test/java/com/a5lab/axion/domain/segment/SegmentCfgControllerTests.java:58]: @A9afon 
* extra vertical line. Could you remove?

File: src/test/java/com/a5lab/axion/domain/segment/SegmentCfgControllerTests.java [hunk 2]
```diff
@@ -23,172 +23,210 @@
...
+    @Test
+    public void shouldShowSegment() throws Exception {
+        final SegmentDto segmentDto = new SegmentDto();
+        segmentDto.setId(10L);
+        segmentDto.setRadar(null);
+        segmentDto.setTitle("My segment");
+        segmentDto.setDescription("My segment description");
+        segmentDto.setPosition(0);
+        segmentDto.setActive(true);
+        segmentDto.setTechnologyBlipList(null);
```
Comment (arufanov) [inline src/test/java/com/a5lab/axion/domain/segment/SegmentCfgControllerTests.java:71]: @A9afon 
* should we set this value? or default value is ok?

**Review by arufanov — CHANGES_REQUESTED** [review]

File: src/test/java/com/a5lab/axion/domain/segment/SegmentCfgControllerTests.java (hunk 2 above)
Comment (A9afon) [reply src/test/java/com/a5lab/axion/domain/segment/SegmentCfgControllerTests.java]: Shouldn't, I'll fix it

File: src/test/java/com/a5lab/axion/domain/segment/SegmentCfgControllerTests.java (hunk 1 above)
Comment (A9afon) [reply src/test/java/com/a5lab/axion/domain/segment/SegmentCfgControllerTests.java:58]: Yes  i agree

File: src/test/java/com/a5lab/axion/domain/segment/SegmentCfgControllerTests.java (hunk 2 above)
Comment (arufanov) [reply src/test/java/com/a5lab/axion/domain/segment/SegmentCfgControllerTests.java]: @A9afon 
* thanks!

File: src/test/java/com/a5lab/axion/domain/segment/SegmentCfgControllerTests.java (hunk 1 above)
Comment (arufanov) [reply src/test/java/com/a5lab/axion/domain/segment/SegmentCfgControllerTests.java:58]: @A9afon 
* thanks!

**Review by arufanov — APPROVED** [review]
@A9afon 
* thanks!

**Pull request merged**